project.go

```diff
@@ -6,15 +6,70 @@ import (
 	"net/url"
 )
 
-// ProjectClient handles communication with the project related methods of the Harbor API.
-type ProjectClient struct {
-	*Client
+// Implementations of ProjectClient handle communication with
```

Comment:
Comments for types should start with the type name. https://github.com/golang/go/wiki/Comments

Reply:
fixed that